src/main/java/net/glowstone/io/anvil/AnvilChunkIoService.java

```diff
@@ -138,6 +143,28 @@ public boolean read(GlowChunk chunk) throws IOException {
             }
         }
 
+        if (levelTag.isList("TileTicks", TagType.COMPOUND)) {
+            List<CompoundTag> tileTicks = levelTag.getCompoundList("TileTicks");
+            for (CompoundTag tileTick : tileTicks) {
+                int tileX = tileTick.getInt("x");
+                int tileY = tileTick.getInt("y");
+                int tileZ = tileTick.getInt("z");
+                String id = tileTick.getString("i");
+                Material material = Material.getMaterial(id);
```

Comment:
IDs are formatted as `provider:material`, e.g `minecraft:flowing_lava`. Also, there is no Bukkit material called `FLOWING_LAVA`. It's just `LAVA`. Stationary lava in vanilla is called `minecraft:lava`, while in Bukkit it's called `STATIONARY_LAVA`.

Don't worry about fixing it, I have already fixed it in my merge conflicts commit.